controllers/activemqartemis_controller.go

```diff
@@ -719,11 +719,11 @@ type inSyncApplyError struct {
 	detail map[string]string
 }
 
-const inSyncWithErrorCause = "some properties resulted in error on pod %s"
+const inSyncWithErrorCause = "some properties from %v resulted in error on pod %s"
```

Comment:
why '%v' and not `%s`?

Reply:
was thinking of putting in the whole projection but settled on just name, %v works well on everything.